Code/Commando/datasafe.h

```diff
@@ -1389,7 +1389,7 @@ uintptr_t DataSafeClass<T>::Get_Type_Code(void)
 	*/
 	static uintptr_t instruction_pointer;
 	instruction_pointer = 0;
-#ifdef _M_IX86
+#if defined(_MSC_VER) && defined(_M_IX86)
```

Comment:
Can't we just removed the asm entirely here... would it really be more performant than the cast?

Reply:
I left this one in datasafe intact because I don't 100% trust the C equivalent.
The datasafe types are used by combate target, which is used by leveledit.

Once we start porting that app, we can find out whether the datasafe types are actually needed. (c++20 might allow us to get rid of them)